graphdatascience/graph_data_science.py

```diff
@@ -43,7 +45,18 @@ def __init__(self, endpoint: Union[str, QueryRunner], auth: Any = None, aura_ds:
 
             driver = GraphDatabase.driver(endpoint, auth=auth, **self._config)
 
-            self._query_runner = self.create_neo4j_query_runner(driver, auto_close=True)
+            neo4j_query_runner = Neo4jQueryRunner(driver, auto_close=True)
+
+            # TODO: Decide what to do if arrow is requested but not present
```

Comment:
Maybe we could _always_ use arrow if it's available? When would you not use it?

Reply:
We do that. This comment refers to the case where the is no flight server I guess